build/quicklink.php

```diff
@@ -80,25 +80,63 @@ function quicklink_enqueue_scripts() {
 	/**
 	 * Filters Quicklink options.
 	 *
+	 * @link https://getquick.link/api/
 	 * @param array {
 	 *     Configuration options for Quicklink.
 	 *
-	 *     @param string[] $urls      Array of URLs to prefetch (override).
-	 *     @param string   $el        CSS selector for the DOM element to prefetch in-viewport links for.
+	 *     @param string   $el        CSS selector for the DOM element to observe for in-viewport links to prefetch.
+	 *     @param int      $limit     The total requests that can be prefetched while observing the $el container.
+	 *     @param int      $throttle  The concurrency limit for simultaneous requests while observing the $el container.
+	 *     @param int      $timeout   Timeout after which prefetching will occur.
+	 *     @param string   $timeoutFn Custom timeout function. Must refer to a named global function in JS.
 	 *     @param bool     $priority  Attempt higher priority fetch (low or high). Default false.
 	 *     @param string[] $origins   Allowed origins to prefetch (empty allows all). Defaults to host for current home URL.
 	 *     @param string[] $ignores   Regular expression patterns to determine whether a URL is ignored. Runs after origin checks.
-	 *     @param int      $timeout   Timeout after which prefetching will occur.
-	 *     @param string   $timeoutFn Custom timeout function.
+	 *     @param string   $onError   Custom error handler. Must refer to a named global function in JS.
+	 *     @param string[] $urls      Array of URLs to prefetch. Deprecated as of plugin version 0.8.0
 	 * }
 	 */
 	$options = apply_filters( 'quicklink_options', $options );
 
+	/**
+	 * Add a deprecation warning for the $options[urls] option.
+	 *
+	 * @since 0.8.0
+	 */
+	if ( isset( $options['urls'] ) && ! empty( $options['urls'] ) ) {
+		_deprecated_argument(
+			__FUNCTION__,
+			'0.8.0',
+			esc_attr( __( 'The "URLs" argument has been deprecated in favor of the dedicated `quicklink_prefetch` filter.', 'quicklink' ) )
```

Comment:
Based on my suggestions:

```suggestion
			esc_attr( __( 'The "urls" argument has been deprecated in favor of the dedicated `quicklink_prefetch()` function.', 'quicklink' ) )
```